beluga_example/config/params.yaml

```diff
@@ -1,101 +1,47 @@
 amcl:
   ros__parameters:
-    # Odometry motion model type.
```

Comment:
I think these are useful for someone creating a new configuration from this example. Why removing them?

Reply:
I reverted the changes here https://github.com/Ekumen-OS/beluga/pull/196/commits/b470a18914370a6a54aaf47457df6603fb9deef5. But I think it's hard to maintain (and remember to edit) the documentation when it's spread all over the repository.